integrated_channels/api/serializers.py

```diff
@@ -23,15 +25,18 @@ class Meta:
         fields = (
             'id',
             'display_name',
+            'channel_code',
```

Comment:
these additions were in most/all of the implementations, so moving it to the common list